package.json

```diff
@@ -182,6 +184,7 @@
     "terser-webpack-plugin": "^1.1.0",
     "traverse": "0.6.6",
     "ts-loader": "4.4.2",
+    "ts-node": "^7.0.1",
```

Comment:
what is this providing for us?

Reply:
Since we can't get rapid feedback from tests, I'm using Quokka for a REPL occasionally.